src/test/resources/packs/simple_objects.yaml

```diff
@@ -0,0 +1,55 @@
+java:
+  Application.java: |
+    package com.example;
+    public class Application {
+      public static void main(String[] args) {
+        new B(new A(42)).bar();
+      }
+    }
+  A.java: |
+    package com.example;
+    public class A {
+        private final int x;
+        public A(int x) {
+            this.x = x;
+        }
+        public int foo() {
+            return x * 2;
+        }
+    }
+  B.java: |
+    package com.example;
+    public class B {
+      private final A a;
+      public B(A a) {
+        this.a = a;
+      }
+      public int bar() {
+        return a.foo() + 1;
+      }
+    }
+eo:
+  Application.eo: |
+    package com.example
+    [] > Application
+      [args...] > main
+        seq > @
+          (b (a 42).new).new.bar
+  A.eo: |
+    package com.example
+    [] > A
+      cage x   
+      [xa] > new
```

Comment:
@volodya-lombrozo `xa` should be a free argument of `A`. You may see in `main` that you do: `(a 42).new` (42 goes to `a`, not to `a.new`). If you leave them as args of `new`, then code in `main` should look like
`(b.new (a.new 42)).bar`. 
`(b (a 42).new).new.bar` seems more correct if we talk about java -> eo mapping

Reply:
@maxonfjvipon What if `A` will have 2 different constructors with different number of parameters? In this case we will have a problem

Reply:
I mean if we leave `xa` as an argument for for `A`

Reply:
@volodya-lombrozo you're right. Let's stick with arguments in `new`. Then example in `main` should look like: `(b.new (a.new 42)).bar`

Reply:
@maxonfjvipon Done